packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -410,7 +410,7 @@ export class ProjectService {
                 // We only allow copying from projects if the user is an admin until we remove the `createProjectAccess` call above
                 if (
                     user.ability.cannot(
-                        'manage',
+                        'create',
```

Comment:
interactive_viewer and above

Reply:
I would have set this to `view` as this check is to check if the user has access to this project. The check for creating a project was before the action to create a project.

Reply:
perhaps we can remove the ability  check here, you're right, the `create` was done already.